tests/integrationtest/t/expression/json.test

```diff
@@ -480,3 +480,6 @@ SELECT JSON_SCHEMA_VALID(CONCAT('{"foo": ',repeat('[', 1000),repeat(']', 1000),'
 # - https://bugs.mysql.com/bug.php?id=106454
 -- error 3853
 SELECT JSON_SCHEMA_VALID('{"properties": {"a": {"exclusiveMinimum": true}}}', '{}');
+
+# TestIssue54027
```

Comment:
```suggestion
# https://github.com/pingcap/tidb/issues/54494
```

A URL might be easier and more future proof. Shouldn't this reference the issue that is closed by this PR instead of the larger one?

Reply:
I'll change it into two lines: 

```
# TestIssue54494
# https://github.com/pingcap/tidb/issues/54494
```